spec/octopus/proxy_spec.rb

```diff
@@ -30,10 +30,10 @@
       config = proxy.config
       expect(config[:adapter]).to eq('mysql2')
       expect(config[:database]).to eq('octopus_shard_1')
-      expect(config[:username]).to eq("#{ENV['MYSQL_USER'] || 'root'}")
+      expect(config[:username]).to eq("#{ENV['MYSQL_USER'] || 'daniel'}")
```

Comment:
👀

Reply:
Forgot to roll back this value. It's done now, thanks for pointing it out!